pyproject.toml

```diff
@@ -0,0 +1,90 @@
+[project]
+name = "earthdata-hashdiff"
+dynamic = ["dependencies", "version"]
+description = 'Python package that hashes content of Earth science data files and compares to reference files.'
+readme = "README.md"
+requires-python = ">=3.10"
```

Comment:
I'm pretty sure all our regression tests use at least Python 3.10. And this way we can avoid issues with `__future__.annotation` for Python 3.9 compatibility.